partner_activity_report/wizard/partner_activity_selection.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Vincent Renaville
+#    Copyright 2014-2015 Camptocamp SA
+#
+#    Author: Damien Crier
+#    Copyright 2015 Camptocamp SA
```

Comment:
The authors should be on two lines, as such:

`#    Author: Vincent Renaville, Damien Crier`
`#    Copyright 2014-2015 Camptocamp SA`